common/src/main/java/com/wynntils/features/user/QuestInfoOverlayFeature.java

```diff
@@ -20,16 +20,30 @@
 import com.wynntils.mc.render.TextRenderTask;
 import com.wynntils.mc.render.VerticalAlignment;
 import com.wynntils.utils.objects.CommonColors;
+import com.wynntils.utils.objects.CustomColor;
 import com.wynntils.wc.utils.scoreboard.quests.QuestInfo;
 import com.wynntils.wc.utils.scoreboard.quests.QuestManager;
+import java.util.ArrayList;
 import java.util.List;
+import net.minecraft.client.resources.language.I18n;
 
 @FeatureInfo(category = "overlays")
 public class QuestInfoOverlayFeature extends UserFeature {
     @OverlayInfo(renderType = RenderEvent.ElementType.GUI)
     private final Overlay questInfoOverlay = new QuestInfoOverlay();
 
     public static class QuestInfoOverlay extends Overlay {
+        private static final List<CustomColor> TEXT_COLORS =
+                List.of(CommonColors.GREEN, CommonColors.ORANGE, CommonColors.WHITE);
+
+        private final List<TextRenderTask> toRender = createRenderTaskList("", "");
+        private final List<TextRenderTask> toRenderPreview = createRenderTaskList(
+                I18n.get("feature.wynntils.questInfoOverlay.overlay.testQuestName") + ":",
```

Comment:
Is this the only one not working, or the other one as well?

Reply:
Do you know at what point I18n is initialized? I'm guessing this is a Minecraft vanilla thing. Can we delay creation of Overlays until after that point? I'm thinking this will not be the only Overlay that will want to have string literals, and it would be nice if we didn't have to jump through hoops to get it.

Reply:
I have no idea, but this is strange overall. I can test it in the evening.

Reply:
Neither of the two I18n literals are working. 

It might also be some environment issue here on my side. I know I have had problems with resources getting placed correctly by IntelliJ. But otoh new textures seem to appear fine, so I don't really think so.

Reply:
Try building the mod and see the localization file in there?

Reply:
I have verified that the strings are in the jar file, so it is not a resource building problem.

I searched the web to try to dig up some "later" init hook that could be called after Fabric's I18n is in place, but failed to find any useful info. :( I've just dropped a question at the Fabric modders discord.

Reply:
Recommendations from the Fabric discord was to either:
1) do translation lookup at render time (I'm not to fond of this), or
2) use a custom resource reload listener, to get a hook back to after i18n has loaded (https://fabricmc.net/wiki/tutorial:custom_resources). 

I'm inclined towards going down route #2, and make a way to do our main init stuff in an "after-resources-are-loaded-init". But I'm not sure what the implications of this is, really.

For the time being, I'm thinking of reverting the I18n bits of this PR to get the rest pushed, and then implement the init-with-resources-loaded, and then getting back to fixing I18n here.

Reply:
(The reload listener needs some additional checks to make sure we only call it once and not on manual resource reloads)